prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -94,12 +285,40 @@ class Hl7Serializer(val metadata: Metadata) {
                 setComponent(terser, element, "MSH-7", formatter.format(report.createdDateTime))
             } else if (element.hl7Field == "MSH-11") {
                 setComponent(terser, element, "MSH-11", processingId)
+            } else if (element.hl7Field != null && element.mapperRef != null && element.type == Element.Type.TABLE) {
+                setComponentForTable(terser, element, report, row)
             } else if (element.hl7Field != null) {
```

Comment:
Added logic to get the value from a table. This is super important to get the county code and LIVD data.